src/GraphQL.Tests/Utilities/NameValidatorTests.cs

```diff
@@ -31,8 +31,19 @@ public void ValidateName_whenNameIsEmpty_throwsArgumentOutOfRange()
             Should.Throw<ArgumentOutOfRangeException>(() => NameValidator.ValidateName(" ", NamedElement.Field));
 
         [Fact]
-        public void ValidateName_whenNameStartsWithReservedCharacters_throwsArgumentOutOfRange() =>
+        public void ValidateName_whenNameStartsWithReservedCharacters_throwsArgumentOutOfRange()
```

Comment:
@Shane32 These tests could be merged.

Reply:
Yeah... I got too much else to do to mess with it, both at work, at home, and here lol.  Go ahead though if you have time.  I really want to finish up these serializer enhancements and write some documentation for it.

I really think the answer benjie gave should be wrong.  But it seems like the graphql spec progresses at a snail's pace.  This is also the second time when the answer given is partially driven by a "reference implementation" even when the implementation is demonstrably incorrect (from my point of view of course).

Reply:
I know, I understand.